requirements.txt

```diff
@@ -6,3 +6,4 @@ pylint==2.9.5
 flake8==3.9.2
 pylint-exit==1.2.0
 pandas==1.4.0
+codecarbon==2.2.1
```

Comment:
This should go in the install_requires section of setup.py -- requirements.txt is just for test/CI deps (and should probably be renamed at some point!)